jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -1004,24 +1004,8 @@ databaseEncryption:
 
 . *Create firewall rules*
 +
-Run the following commands to create a firewall rule for VM service in GKE:
-+
-NOTE: If you have used auto-mode, you can find the Nomad clients CIDR based on the region by referring to the https://cloud.google.com/vpc/docs/vpc#ip-ranges[table here].
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-internal-nomad-fw" --network "<network>" --action allow --source-ranges "0.0.0.0/0" --rules "TCP:22,TCP:2376"
-----
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-internal-k8s-fw" --network "<network>" --action allow --source-ranges "<clusterIpv4Cidr>" --rules "TCP:22,TCP:2376"
-----
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-external-fw" --network "<network>" --action allow --rules "TCP:54782"
-----
+
+External VMs need the networking rules described in [Hardening your Cluster](https://circleci.com/docs/server/installation/hardening-your-cluster/#external-vms)
```

Comment:
@rosieyohannan i will defer to you on the best was to build this link, i suspect there is some internal path referencing that would be prefered